### PR TITLE
Split equation on line break for MathML

### LINKED ISSUE #179: Investigate line-breaks in Asciimath (and MathML and OOXML)
https://github.com/metanorma/metanorma-standoc/issues/111 has been stuck for 4 years, because we had no good mechanism to input line-breaks in Asciimath that we could guarantee would be respected in MathML and OOXML.

Now that we have Plurimath, I'd like for us to revisit that, and see how far we can get with implementing `\` as Asciidoctor's extension to Asciimath realising line break.

See the standoc ticket for discussion.

### LINKED ISSUE #201: Strange rendering in matrices written with latexmath
*In relation to https://github.com/metanorma/iso-10303-detached-docs/pull/233*

### Sample 1

```
[latexmath%unnumbered]
++++
   \left(
\begin{array}{cccccc}
{\displaystyle \frac{1}{E}} & -{\displaystyle \frac{\nu}{E}} & -{\displaystyle \frac{\nu}{E}} & 0 & 0 & 0 \\
     & {\displaystyle \frac{1}{E}}  & -{\displaystyle \frac{\nu}{E}} & 0 & 0 & 0 \\
     & & {\displaystyle \frac{1}{E}} & 0 & 0 & 0 \\
     & & &  {\displaystyle \frac{1}{G}}  & 0 & 0 \\
     & \mbox{symmetric} &  &  & {\displaystyle \frac{1}{G}} & 0 \\
     & & & & & {\displaystyle \frac{1}{G}}
\end{array}
\right)
++++
```

renders:

PDF:
eq-1-pdf

HTML:
eq-1-html

Original:
eq-1-original

### Sample 2

```
[latexmath%unnumbered]
++++
\begin{array}{cc}
\left(    \begin{array}{ccccccc}
 & & & & & & \\
 & +k & & & & -k  \\
 & & & & & & \\
 & & & & & & \\
 & & & & & & \\
 & -k & & & & +k
    \end{array}  \right)   &
  \begin{array}{cc}
   & \\
   \cdots & \mbox{degree of freedom 1, node 1} \\
   & \\
   & \\
   & \\
   \cdots & \mbox{degree of freedom 2, node 2}
  \end{array}      \\
      & \\
 \begin{array}{cccccc}
    \vdots & & & & & \vdots
     \end{array}  & \\
     \begin{array}{cccc}
& \mbox{degree of}  & \mbox{degree of}  & \\
& \mbox{freedom 1,} & \mbox{freedom 2,} & \\
& \mbox{node 1} & \mbox{node 2}  &
    \end{array}    &
\end{array}
++++

```

renders

PDF:
eq-2-pdf

HTML:
eq-2-html

Original:
eq-2-original

### To reproduce:

Use this sample file: 
[equations.zip](https://github.com/plurimath/plurimath/files/13398871/equations.zip)

### PR DESCRIPTION
**Pull request by suleman-uzair**
This PR will add support for equation split on line break for MathML.

closes #201

### CODE REVIEW

**Comment by suleman-uzair** [comment]
> @suleman-uzair please help merge this when ready since approved by @opoudjis . Thanks!

@ronaldtse , I will rebase it after verifying the code quality and then I'll merge it.

File: lib/plurimath/math/function/oint.rb [hunk 1]
```diff
@@ -25,28 +25,31 @@ def to_latex
...
-                     else
-                       parameter_one ? "sub" : "sup"
-                     end
-          msubsup_tag = Utility.ox_element("m#{tag_name}")
-          Utility.update_nodes(msubsup_tag, value_array)
-          return msubsup_tag if parameter_three.nil?
+          if parameter_one || parameter_two
+            value_array = [mo_tag]
+            value_array << parameter_one&.to_mathml_without_math_tag
+            value_array << parameter_two&.to_mathml_without_math_tag
```
Comment (HassanAkbar) [inline lib/plurimath/math/function/oint.rb:35]:
```suggestion
            value_array = [mo_tag, parameter_one&.to_mathml_without_math_tag, parameter_two&.to_mathml_without_math_tag]
```

Comment (HassanAkbar) [reply lib/plurimath/math/function/oint.rb:35]: Also, look for occurrences like this in the PR.

**Review by HassanAkbar — COMMENTED** [review]
Overall the PR looks good 👍. Just added some comments related to code quality.

**Pull request merged**